admin/class-purger.php

```diff
@@ -461,7 +461,11 @@ protected function do_remote_get( $url ) {
 		 */
 		do_action( 'rt_nginx_helper_before_remote_purge_url', $url );
 
-		$response = wp_remote_get( $url );
+		$request_args = array(
+			'blocking' => false,
```

Comment:
Doing this will break the logging functionality and any actions that other plugins are performing on the hook `rt_nginx_helper_after_remote_purge_url` since a non blocking request initially starts with status 0, meaning the request is still processing, instead of actual status code like 200 or 404.